README.md

```diff
@@ -29,6 +29,7 @@ This is considered an intermediate learning resource, and should typically be re
 | [Shipping](use-cases/shipping-v2)                                | Python                   | -                                                                                                                         |
 | [Solicitations](use-cases/solicitations)                         | C# / Python              | -                                                                                                                         |
 | [Vendor Direct Fulfillment](use-cases/vendor-direct-fulfillment) | C#                       | [Vendor Direct Fulfillment Samples](https://developer-docs.amazon.com/sp-api/docs/code-samples#vendor-direct-fulfillment) |
+| [LWA Rotation](use-case/lwa-rotation)                            | Java                     | -                                                                                                                         |
```

Comment:
Redirect is broken. Minor fix here `use-cases/lwa-rotation` cases need to be plural.